numba/np/arraymath.py

```diff
@@ -155,7 +155,7 @@ def create_full_slice():
     return function_sig, codegen
 
 
-#----------------------------------------------------------------------------
+# ----------------------------------------------------------------------------
```

Comment:
It's good to try and avoid cosmetic changes in PRs that are about functionality, this makes it easier for reviewers as the change set is smaller, it also somewhat prevents PR `X` making a change one way and PR `Y` making it another.

Reply:
Thank you.  I won't repeat this

Reply:
Thanks, and no worries, it's just a convention Numba has to try and reduce review time. If you want to do cosmetic changes then feel free to open PRs for just those changes.